source/Account.cpp

```diff
@@ -268,10 +273,13 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	{
 		if(salariesPaid)
 			out << creditString(salariesPaid) << " in crew salaries"
-				<< ((mortgagesPaid || finesPaid || maintenancePaid) ? " and " : ".");
+				<< ((mortgagesPaid || finesPaid || maintenancePaid || sharesPaid) ? " and " : ".");
 		if(maintenancePaid)
 			out << creditString(maintenancePaid) << "  in maintenance"
-				<< ((mortgagesPaid || finesPaid) ? " and " : ".");
+				<< ((mortgagesPaid || finesPaid || sharesPaid) ? " and " : ".");
+		if(sharesPaid)
+			out << creditString(sharesPaid) << " in profit shares"
+				<< ((finesPaid || mortgagesPaid) ? " and " : ".");
```

Comment:
```suggestion
				<< ((mortgagesPaid || finesPaid || profitSharesPaid) ? " and " : ".");
		if(sharesPaid)
			out << creditString(profitSharesPaid) << " in profit shares"
				<< ((finesPaid || mortgagesPaid) ? " and " : ".");
```

Reply:
I added the word `outstanding` to help communicate that the crew would really prefer if we paid them their profit shares right away.